packages/v4/src/content/accessibility/skip-to-content/skip-to-content.md

```diff
@@ -0,0 +1,16 @@
+---
+id: Skip to content
+section: components
+---
+
+A **skip to content** is a component that allows screen reader and keyboard users to bypass navigation rather than tabbing through it.
+
+**Keyboard users** should be able to find the skip to content using **Tab** at the start of a page with navigation. They should be able to select the skip to content using **Enter**.
+
+**Screen reader users** should also be able to navigate to skip to content using **Tab** and select it to skip past the navigation of a page.
+
+A skip to content does not trap tab focus. It is the first tabbable element on a page, but can be tabbed past and into the navigation of a page if the user prefers.
+
+## To make skip to content accessible:
+- Make sure the skip to content can be accessed with **Tab** 
+- Give the target of the skip to content link a tabindex of -1. Internet Explorer and Chrome require this attribute to make this component work consistently.
```

Comment:
I don't think this one (the tabindex) is an absolute requirement? It doesn't seem to be included in the [WCAG techniques](https://www.w3.org/TR/WCAG20-TECHS/G1.html) or the example they include. Curious of your thoughts on this one though or which resource you found. Maybe it's totally fine here but thought I'd see which resource you're referencing for this. 

Looking at the skip to content example, I think the big thing is just making sure it's hooked up correctly with a proper href like `href="#main-content"` on the SkipToContent and having it point to the id of the main content. It's a pretty easy set up if they're using the Page component, and there is the possibility that they'll have to add context to the href if they have any query parameters in the URL. That might be something we should mention as well. For example, when I [added the skip to content to OpenShift](https://github.com/openshift/console/pull/6596/files), it was a fairly straightforward fix, but it needed context in the href.

Reply:
https://accessibility.oit.ncsu.edu/it-accessibility-at-nc-state/developers/accessibility-handbook/mouse-and-keyboard-events/skip-to-main-content/

This is where I found that info; it might be outdated though, so I can remove it!

Reply:
Based on our discussion, I think it would make sense to add a bullet point that says that if you're using the Page component with the Skip to Content, we add a `tabindex="-1"` for you, but if you are using the Skip to Content on its own, you will need to add it to the target (you can keep some of your previous wording). 

I realized in our meeting that the reason I didn't need to add the tabIndex to OpenShift is that it's included when using the Page and Skip to Content together, which I did. 🙂